src/services/types.ts

```diff
@@ -1501,6 +1503,12 @@ export interface OutliningSpan {
     kind: OutliningSpanKind;
 }
 
+export interface MapCodeDocumentMapping {
+    fileName?: string;
+    focusLocations?: TextSpan[][];
+    contents: string[];

```

Comment:
I'd rather not change the protocol any further if I can avoid it, since that's already being used by a number of teams and has pretty high overhead for alignment.

Reply:
Who is using this without changes in tsserver..

Reply:
C# is already using the protocol for their own codemapper, so is Razor. Other languages are following suit.

Reply:
For me, changing it in mapCode.ts is the important part because it makes the code easier to read.

Reply:
I think we should keep the name, because consistency is more important to me in this case.